backend/openapi.json

```diff
@@ -757,9 +757,13 @@
         "description": "Response structure for getting data entry of polling station results",
         "required": [
           "data",
+          "client_state",
           "validation_results"
         ],
         "properties": {
+          "client_state": {
+            "type": "object"
```

Comment:
I guess fine for now but it would be nice if we could type this.

Reply:
@cikzh also mentioned this on Mattermost. For now it's fine because the backend doesn't need the client state, and the type in the frontend is more specific than what  we have in OpenAPI (using `FormSectionID` instead of just `string`).